pom.xml

```
@@ -194,6 +195,17 @@
             <version>2.0.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib</artifactId>
```

Inkluderer visst "stream extensions and duration extensions"

```suggestion
            <artifactId>kotlin-stdlib-jdk8</artifactId>
```